client/src/API.ts

```diff
@@ -0,0 +1,64 @@
+import { Dispatch, SetStateAction } from "react";
+import { castCommand, extractTextContent, extractTextStyles } from "./Command";
+import { TextStyles } from "./components/Typewriter";
+
+interface SGProps {
+  setConversation: Dispatch<SetStateAction<string[]>>;
+  setGameID: Dispatch<SetStateAction<string>>;
+  setStyles: Dispatch<SetStateAction<TextStyles>>;
+  isMounted: () => boolean;
+}
+
+export const startGame = async ({
+  setConversation,
+  setGameID,
+  setStyles,
+  isMounted,
+}: SGProps) => {
+  const response = await fetch("http://127.0.0.1:5000/start");
+
+  if (!response.ok || !isMounted()) {
+    return;
+  }
+
+  const data = await response.json();
+  const command = castCommand(data.response);
+  const styles = extractTextStyles(command);
+  if (data.message && typeof data.message === "string") {
+    setConversation((prev) => [...prev, data.message]);
+    setGameID(data.game_id);
+    setStyles(styles);
+  }
+};
+
+export async function playGame(
```

Comment:
This doesn't play the whole game, just the server/game's "turn"? We should come up with a name for this... how about [ply](https://en.wikipedia.org/wiki/Ply_(game_theory))?